base/pprof.cpp

```diff
@@ -12,6 +12,8 @@ PProf::PProf(std::string const & path)
 {
 #if defined(USE_PPROF)
   ProfilerStart(path.c_str());
+#else
+  UNUSED_VALUE(path);
```

Comment:
AFAIK, Qt, which is one of examples of more or less readable C++ code, uses a similar macro to suppress unused arguments when they are unavoidable (e.g., when overriding certain interfaces). Does it make sense to use it everywhere instead of commenting out arguments in the declaration?

Reply:
What about building for Android/iOS? We don't want to include Qt libs there, don't we?